app/presenters/two_factor_authentication/set_up_backup_code_selection_presenter.rb

```diff
@@ -11,7 +11,11 @@ def label
     end
 
     def info
-      t('two_factor_authentication.two_factor_choice_options.backup_code_info')
+      # i18n-tasks-use t('numbers_spelled_out.10')
+      t(
+        'two_factor_authentication.two_factor_choice_options.backup_code_info',
+        count: t("numbers_spelled_out.#{BackupCodeGenerator::NUMBER_OF_CODES}"),
+      )
```

Comment:
I think this might be a reason to consider a gem like https://github.com/18F/identity-idp/pull/10970#discussion_r1686974895, because then can rely on that gem's translations, and we don't need the `i18n-tasks-use` comment

```suggestion
      t(
        'two_factor_authentication.two_factor_choice_options.backup_code_info',
        count: BackupCodeGenerator::NUMBER_OF_CODES.to_words,
      )
```

Reply:
we might have to augment with like a
```
def readable_number(number)
  case I18n.locale
  when :en
    number.to_words
  when :zh
    number.to_s
  # etc
end
```

Reply:
Yeah, the two issues I had with that approach are:

- The intent here is not to necessarily always spell out the number, but to have it translated to how it'd typically be written into a sentence (per your second comment)
- `numbers_and_words` doesn't have translations for every language ([source](https://github.com/kslazarev/numbers_and_words/tree/master/lib/numbers_and_words/translations)), currently lacking `zh`, and unsure what future language support might look like

With the approach in your second comment, it kinda addresses both points, but dunno how well that works long-term, or if it's really any simpler than what I ended up doing here.

Reply:
I think it would be valuable to avoid updating the translation YML files every time a config number changes! And being able to pull in a gem that covers most of our languages is really nice for that. And it seems moot that ZH isn't supported by the gem since it seems our approach is to use digits.

And if in the fugure we do need to use chinese digits, I think the system is straightforward enough that we could implement it ourselves and contribute upstream to the gem

https://www.edhelper.com/ChineseNumbersIntro.htm#:~:text=The%20Chinese%20numeration%20system%20has,as%20other%20multiples%20of%20ten.

Reply:
Gave it a shot in 03bc73eaa3. Let me know what you think

Reply:
If we're not changing the numbers, do we still want to include it?

Reply:
We're still changing the content, and IMO the original implementation should have interpolated the count to begin with.

Reply:
Big fan! Thanks

Reply:
One thing -- the `numbers_and_words` seems to support genders for numbers but I'm not seeing that option for humanize`? Probably fine for now, but definitely curious to see if that may be relevant or us in the future

Reply:
Yeah, could save us some work in the future. `numbers_and_words` also has more languages and easier way to check that a language is supported. Swapped in 6100eefa36.